tests/test_indefinite_freeze_attack.py

```diff
@@ -401,5 +401,101 @@ def test_with_tuf(self):
         ' Freeze attack successful.')
 
 
+
+
+    # Test 3 Begin:
+    #
+    # Serve the client expired Snapshot.  The client should reject the given,
+    # expired Snapshot and the locally trusted one, which should now be out of
+    # date.  After the attack, attempt to re-issue a valid Snapshot to verify
+    # that the client is able to recover from the indefinite freeze attack via
+    # the snapshot metadata.
```

Comment:
I think it will be a bit confusing to future readers to refer to this as an indefinite freeze attack here without context (since there is nothing that obviously represents an attack in the test below). Perhaps:

```Python
# Serve the client expired Snapshot.  The client should reject the given,
# expired Snapshot and the locally trusted one, which should now be out of
# date.
# After the attack, attempt to re-issue a valid Snapshot to verify that
# the client is still able to update. (A bug previously caused snapshot
# expiration or replay to result in an indefinite freeze.)
```

You could also point to the particular issue, #736, in the comment, though that's probably less important. If that looks good, I can add the commit.

Reply:
I like the comment that you added, but we should also point to #736 so the reader can understand how the bug was caused.

I'd appreciate you adding the commit, if you are not busy.